### ISSUE TITLE
Issue when importing .ris into mendeley

### ISSUE THREAD

**Issue by Mailinnia** [nealhaddaway/citationchaser#28]
Hi,

I'm having issues when importing the resulting RIS file from citationchaser into mendeley, as the author order is wrong.

The issue seems to be the AU field in the RIS file.
Current RIS:
```
TY  - JOUR
AU  - Sánchez-Cordón, Pedro J.; Montoya, Maria; Reis, Ana Luisa; Dixon, Linda K.
TI  - African swine fever: A re-emerging viral disease threatening the global pig industry
...
```
Mendeley reads it correctly when it's the following format:
```
TY  - JOUR
A1  - Sánchez-Cordón, Pedro J.
A1  - Montoya, Maria
A1  - Reis, Ana Luisa
A1  - Dixon, Linda K.
TI  - African swine fever: A re-emerging viral disease threatening the global pig industry
...
```

Would it be possible to download in this format?

**Comment by nealhaddaway** [comment]
Great point - but that's a pretty substantial rewrite, so not something I'll likely be able to do in the foreseeable future, sadly... Help greatly appreciated here anyone...?

For anyone able to help, essentially, the authors field needs to be changed from collapsing authors separated by ';' to collapsing separated by '\nAU = ' (but not where there is a single authors). Also a note to myself in case I have time in spring.

**Comment by DrMattG** [comment]
Closed with #33

**Comment by DrMattG** [comment]
@Mailinnia Can you see if this works for you now?
